Source/KeyStorage/PrivateKeyStorage/PrivateKeyEntry.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PrivateKeyEntry.swift
+//  VirgilSDK
+//
+//  Created by Eugen Pivovarov on 2/13/18.
+//  Copyright © 2018 VirgilSecurity. All rights reserved.
+//
+
+import Foundation
+import VirgilCryptoAPI
+
+@objc(VSSImportedKeyEntry) public class ImportedKeyEntry: NSObject {
+    @objc public let privateKey: PrivateKey
+    @objc public let meta: [String: String]?
+
+    public init(privateKey: PrivateKey, meta: [String: String]? = nil) {
```

Comment:
@Ogerets Why not @objc?

Reply:
@SanjoDeundiak Maybe we need to make it internal to not allow clients create it? It opens a possibility for substitution PrivateKeyEntry passing through the functions for some reason. Is it good?

Reply:
@Ogerets no reasons to forbid that